lib/instream/connection/query_runner_v2.ex

```diff
@@ -144,6 +145,34 @@ defmodule Instream.Connection.QueryRunnerV2 do
     result
   end
 
+  @doc """
+  Executes `:delete` predicate.
+  """
+  @spec delete(map(), Keyword.t(), module) :: any
+  def delete(points, opts, conn) do
```

Comment:
We should probably rename the `points` parameter everywhere to something like `payload` (like in the connection behaviour). After all we are performing a specified delete request and not handling any "points" directly.

Reply:
Renamed. Thank you.